pangeachat/client
# fix word sticking together, finally 🤯

Testing steps to ensure it's fixed:

- Obtain a message that has the following format "A bunch of text, 🥲 then emojis in between, 🤯 then more texts". A reliable method to achieve this kind of message is as follow:
  - Create chat group
  - Invite bot in discussion mode
  - React the ⏩ icon in bot join message
  - It should send a message with the desired format
- Click on the message to trigger practice activities
- The message should not be sticking together
